### PR TITLE
Limit the size of data printed by the debugger

### PR DESCRIPTION
**Pull request by Malabarba**
Goes with https://github.com/clojure-emacs/cider/pull/1191

### CODE REVIEW

File: src/cider/nrepl/middleware/debug.clj [hunk 1]
```diff
@@ -117,12 +117,19 @@
   "Bound by the `breakpoint` macro to the local &env."
   {})
 
+(defn pr-short
+  "Like `pr-str` but limited in length and depth."
+  [x]
+  (binding [*print-length* 6
```
Comment (bbatsov) [inline src/cider/nrepl/middleware/debug.clj]: Isn't it better to give some control to the users to configure this? E.g. we can pass some data from cider. Or maybe this is just going to be overkill...

Comment (Malabarba) [inline src/cider/nrepl/middleware/debug.clj]: Pushed an updated that lets us specify these values when the debugger is initialized. I'll make the cider.el counterpart soon.

**Comment by Malabarba** [comment]
Finally, tests are passing.

**Comment by bbatsov** [comment]
:+1:

**Pull request merged**